elfpy/agents/agent.py

```diff
@@ -254,13 +250,25 @@ def update_wallet(self, wallet_deltas: wallet.Wallet, market: Market) -> None:
                 )
                 self.wallet[key].amount += value_or_dict.amount
             # handle updating a dict, which have mint_time attached
+            elif key == "borrows":
+                if value_or_dict:  # could be empty
+                    self._update_borrows(value_or_dict)
             elif key == "longs":
                 self._update_longs(value_or_dict.items())
             elif key == "shorts":
                 self._update_shorts(value_or_dict.items())
             else:
                 raise ValueError(f"wallet_key={key} is not allowed.")
 
+    def _update_borrows(self, borrow_summary: wallet.Borrow) -> None:
```

Comment:
this looks good for now, i assume we'll be updating all fields in a borrow though, not just the borrow_amount?

Reply:
yes eventually